.devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm not sure what this is, and I don't use it myself, so I'd rather not have it in the project.

Reply:
This is the file that enables this: https://code.visualstudio.com/docs/remote/containers
Basically an easy way to have a development environment with everything ready to go by opening it with Vscode